src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,14 +35,20 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        String email = authentication.getName();
+        User user = userService.findByEmail(email)
```

Comment:
```suggestion
        User user = userService.findByEmail(authentication.getName())
```
make the code consistent